carma-messenger-ui/website/widgets/truckInspection/widget.js

```diff
@@ -313,6 +318,22 @@ CarmaJS.WidgetFramework.truckInspection = (function () {
 
             console.log("truckInspectionSafetyLogPerVIN element is created");
     }
+
+    var IsValidVINNumber = function(vin_number)
+    {
+        let isValid = false;
+        console.log("Check VIN: " + vin_number);
+        let valid_truck_vin_numbers = CarmaJS.Config.getValidTruckVINNumber();
+        valid_truck_vin_numbers.forEach((valid_vin)=>{
+            console.log("Valid VIN: " + valid_vin);
+            if(valid_vin.trim() == vin_number.trim()){
```

Comment:
If you've trimmed on line 23, i think what you want to check is the actual valid_vin "as is", without having to trim it again to compare "as is" the "value" you have stored and using for the jpg name.

Reply:
Addressed